R/data_color.R

```diff
@@ -147,6 +147,15 @@ data_color <- function(data,
   columns <- rlang::enquo(columns)
   resolved_columns <- resolve_vars(var_expr = !!columns, data = data)
 
+  # Get the sequence of row indices for the table
+  rows <- seq_len(nrow(data_tbl))
+
+  data_color_styles_tbl <-
+    dplyr::tibble(
+      locname = character(0), grpname = character(0), colname = character(0),
+      locnum = numeric(0), rownum = integer(0), colnum = integer(0), styles = list()
+    )
+
   for (column in resolved_columns) {
 
     data_vals <- data_tbl[[column]]
```

Comment:
Should be `data_tbl[rows, column]`

Reply:
Because `data_tbl` is a tibble, the above would subset to a one-column tibble. Because we want of vector of values, and still want to incorporate `rows`, I went with `data_tbl[[column]][rows]`.